dev/drivers/scripts/cam/plots/jevs_cam_radar_plots.sh

```diff
@@ -1 +1,79 @@
-sample
+#!/bin/bash
+#PBS -N jevs_cam_radar_plots_00
+#PBS -j oe
+#PBS -S /bin/bash
```

Comment:
Just a nitpick ... b/c you have the job card `#PBS -S /bin/bash` you do not need to state `#!/bin/bash` at line 1 of the driver scripts.

Reply:
just removed that line from both driver scripts included here. I'll remove that line from the driver scripts included in PR #161. I'll include the update on that PR.